### PR TITLE
tests need to be skipped for topology

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

**Comment by jassigill2000** [comment]
waiting for topology test run on consumer with ssl enabled to finish, then we can merge this.

File: scripts/irods/test/test_iticket.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def tearDown(self):
     def test_iticket_bad_subcommand(self):
         self.admin.assert_icommand('iticket badsubcommand', 'STDOUT_SINGLELINE', 'unrecognized command')
 
+    @unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, "Skip for topology testing from resource server")
```
Comment (trel) [inline scripts/irods/test/test_iticket.py:32]: this file needs more
```
from .. import test
```
near the top

File: scripts/irods/test/test_iunreg.py [hunk 2]
```diff
@@ -41,6 +41,7 @@ def tearDown(self):
             os.unlink(self.local_data_path)
         super(Test_Iunreg, self).tearDown()
 
+    @unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, "Skip for topology testing from resource server")
```
Comment (trel) [inline scripts/irods/test/test_iunreg.py:44]: add to top of file...
```
from .. import test
```

File: scripts/irods/test/test_iunreg.py [hunk 3]
```diff
@@ -73,7 +75,8 @@ def test_iunreg_non_vault(self):
         self.assertTrue(
             os.path.exists(self.local_data_path),
             msg='Data missing from local file after unregister:[{}]'.format(self.local_data_path))
-
+
```
Comment (trel) [inline scripts/irods/test/test_iunreg.py:78]: fix whitespace

File: scripts/irods/test/test_iunreg.py [hunk 4]
```diff
@@ -138,6 +142,7 @@ def test_iunreg_replica_number(self):
             os.path.exists(self.data_in_repl_vault_path),
             msg='Data missing from vault after unregister:[{}]'.format(self.data_in_repl_vault_path))
 
+    @unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, "Skip for topology testing from resource server")
```
Comment (trel) [inline scripts/irods/test/test_iunreg.py:145]: fix trailing whitespace

File: scripts/irods/test/resource_suite.py [hunk 5]
```diff
@@ -519,6 +519,7 @@ def test_local_iput_physicalpath_no_permission(self):
         if os.path.exists(datafilename):
             os.unlink(datafilename)
 
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "Skip for Topology Testing")
```
Comment (trel) [inline scripts/irods/test/resource_suite.py:522]: @alanking why would this need to be skipped?   we're not looking in the vault...

File: scripts/irods/test/test_iticket.py [hunk 6]
```diff
@@ -29,6 +31,7 @@ def tearDown(self):
     def test_iticket_bad_subcommand(self):
         self.admin.assert_icommand('iticket badsubcommand', 'STDOUT_SINGLELINE', 'unrecognized command')
 
+    @unittest.skipIf(test.settings.TOPOLOGY_FROM_RESOURCE_SERVER, "Skipping it till the issue 4931 is fixed")
```
Comment (trel) [inline scripts/irods/test/test_iticket.py:34]: let's say "Skipping for now - needs investigation - issue 4931"

File: scripts/irods/test/test_iunreg.py [hunk 7]
```diff
@@ -59,6 +62,7 @@ def test_iunreg_from_vault(self):
...
```
Comment (trel) [inline scripts/irods/test/test_iunreg.py:65]: trailing whitespace

**Pull request merged**